assets/css/sensei-course-theme/layout.scss

```diff
@@ -1,13 +1,17 @@
 :root {
-	--header-height: var(--focus-header-height, 75px);
+	--header-height: 75px;
```

Comment:
Title wrapping is broken for me on this branch, it breaks into a new line even with ellipsis working:

<img width="397" alt="image" src="https://user-images.githubusercontent.com/176949/151461556-103f9b21-9e3f-4432-84e0-92e82222039f.png">

_(Not related to the line I commented this on, just wanted to start a thread)_

Reply:
Interesting! For mobile, it should hide the progress based on [our styles](https://github.com/Automattic/sensei/blob/74225c7e3ce1c3877c56366552b54dffa9c27646/assets/css/sensei-course-theme/mobile.scss#L55-L57). I couldn't reproduce it on my end. Could you inspect on your end for me to understand what happened there? Maybe the block was moved to some part where the class didn't get?

Reply:
This is when the sidebar is open. But the problem is even when it's closed, the title disappears because it breaks into a new line. I'll look into it.

Reply:
Ah, okay! I got it! I can see it now! =)

Reply:
It should be fixed now with this commit: https://github.com/Automattic/sensei/pull/4684/commits/0f4bfa73365659c5c1481e0b2ab684878d346d45

Do you remember why there was that property there? Maybe we just forgot to remove it before some change?

Reply:
I think it's for wrapping the course progress text block to a new line: we show that when the sidebar is open:

<img width="547" alt="image" src="https://user-images.githubusercontent.com/176949/151566901-61ade82e-57d4-468b-a1c2-91f48d2a464b.png">